packages/react-composites/tests/browser/call/hermetic/RoomsCallScreen.test.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { buildUrlWithMockAdapter, defaultMockCallAdapterState, test } from './fixture';
+import { expect } from '@playwright/test';
+import { dataUiId, stableScreenshot, waitForSelector } from '../../common/utils';
+import { IDS } from '../../common/constants';
+
+test.describe('Rooms CallScreen tests for different roles', async () => {
+  test('All CallControls are enabled for Presenter', async ({ page, serverUrl }) => {
+    const initialState = defaultMockCallAdapterState();
```

Comment:
should we be adding any remote participants in these tests? what does this look like if you are a consumer and one of the remote participants is the organizer for example?

Reply:
I have added the remote participants for the tests for consumer, but we cannot control the role for remote participant as it is passed as a prop to call composite as of now. Let me know if you have any thoughts on that.